src/ResourceManager/DevTestLabs/Commands.DevTestLabs/Commands/DevTestLabsCmdletBase.cs

```diff
@@ -59,6 +59,9 @@ internal IDevTestLabsClient DataServiceClient
                 {
                     _dataServiceClient = AzureSession.Instance.ClientFactory.CreateArmClient<DevTestLabsClient>(DefaultContext,
                         AzureEnvironment.Endpoint.ResourceManager);
+
+                    WriteInformation(new InformationRecord($"Resource Group = {ResourceGroupName}", nameof(DevTestLabsCmdletBase)));
```

Comment:
Cmdlets should be able to run on PowerShell 3.  Please write to the Verbose or Progress streams instead